api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/rules/validationrules/impl/AdultOnlineZeroCourseHistoryRule.java

```diff
@@ -58,8 +53,10 @@ public List<SdcSchoolCollectionStudentValidationIssue> executeValidation(Student
         final List<SdcSchoolCollectionStudentValidationIssue> errors = new ArrayList<>();
 
         boolean isAdult = DOBUtil.isAdult(studentRuleData.getSdcSchoolCollectionStudentEntity().getDob());
+        String schoolType = studentRuleData.getSchool().getFacilityTypeCode();
+        boolean isOnline = Objects.equals(schoolType, "DIST_LEARN") || Objects.equals(schoolType, "DISTONLINE");
```

Comment:
Please use the constants for these string values DIST_LEARN. DISTONLINE. FacilityTypeCode.